pages/chain/beam-ecosystem/sphere.mdx

```diff
@@ -19,12 +19,25 @@ Explore Beam’s native marketplace at [sphere.market](https://sphere.market).
 
 ### What networks support Sphere?
 
-Apart from Beam, Sphere is accessible on Avalanche, Ethereum, and Polygon networks.
+Apart from Beam, Sphere is accessible on the Immutable zkEVM network.
 
 ### Are there any fees associated with Sphere transactions?
 
-Yes, selling an asset on Sphere incurs a standard fee of 2.5% of the transaction amount, charged in the currency used for the sale. Additionally, there might be royalties set by the creator and a gas fee to initiate the transaction. Learn more about [Sphere Fees](https://sphere.market/fees).
+Yes, when selling an asset on Sphere, the following fees occur:
+  - Trading fee: This is a standard 2% of the paid amount charged in the currency used to make the sale. 
+  - Royalties: This is a variable percentage that is set by the creator.
+  - Gas fees: When selling an NFT, you need to approve the smart contracts to interact with your assets, the actual transaction is covered by the buyer of the NFT.
```

Comment:
You can also mention the fact that cancelling a listing on Beam also incurs gas fees. On the Immutable zkEVM chain, we support gassless cancellations.